en/docs/Learn/DesignAPI/CreateAPI/generate-rest-api-from-soap-backend.md

```diff
@@ -40,50 +40,52 @@ Follow the instructions below to generate REST APIs in WSO2 API Manager for an e
 
-5.  Navigate to the **API Definition** tab and click on **Edit** to modify the open API Definition of the API.
-    ![](../../../assets/img/Learn/api-definition-of-generated-rest-api-from-soap-backend.jpg)
-
-6.  The generated API resources are added to the API as shown below.
-    ![](../../../assets/img/Learn/generated-resources-of-soap-backend.jpg)
+5.  Click **API Definition** and click **Edit** to modify the open API Definition of the API.
+     ![]({{base_path}}/assets/img/Learn/api-definition-of-generated-rest-api-from-soap-backend.jpg)
+    
+     The generated API resources are added to the API, as shown below.
+     ![]({{base_path}}/assets/img/Learn/generated-resources-of-soap-backend.jpg)
 
 7.  Click on a resource to view the In and Out sequences of the API.
-    [![](../../../assets/img/Learn/in-out-sequences-of-generated-rest-api.jpg)](../../../assets/img/Learn/in-out-sequences-of-generated-rest-api.jpg)
-
-8.  The following sample shows the generated API In-sequence for a POST method.
-
-    ``` java
-           <header description="SOAPAction" name="SOAPAction" scope="transport" value="http://ws.cdyne.com/PhoneVerify/query/CheckPhoneNumber"/>
-            <property name="REST_URL_POSTFIX" scope="axis2" action="remove"/>
-            <property expression="json-eval($.CheckPhoneNumber.LicenseKey)" name="req.var.CheckPhoneNumber.LicenseKey"/>
-            <property expression="json-eval($.CheckPhoneNumber.PhoneNumber)" name="req.var.CheckPhoneNumber.PhoneNumber"/>
-
-
-            <payloadFactory description="transform" media-type="xml">
-            <format>
-            <soapenv:Envelope xmlns:soapenv="http://www.w3.org/2003/05/soap-envelope" xmlns:web="http://ws.cdyne.com/PhoneVerify/query">
-            <soapenv:Header/>
-            <soapenv:Body>
-               <web:CheckPhoneNumber xmlns:web="http://ws.cdyne.com/PhoneVerify/query">
-            <web:LicenseKey>$1</web:LicenseKey>
-            <web:PhoneNumber>$2</web:PhoneNumber>
-            </web:CheckPhoneNumber>
-
-            </soapenv:Body>
-            </soapenv:Envelope>
-            </format>
-            <args>
-               <arg evaluator="xml" expression="get-property('req.var.CheckPhoneNumber.LicenseKey')"/>
-            <arg evaluator="xml" expression="get-property('req.var.CheckPhoneNumber.PhoneNumber')"/>
-
-            </args>
-            </payloadFactory>
-            <property description="messageProperty" name="messageType" scope="axis2" type="STRING" value="application/soap+xml"/>
-    ```
-
-    The incoming JSON message parameters are stored using properties. The SOAP payload needed for the backend is generated using a payload factory mediator.
+    [![]({{base_path}}/assets/img/Learn/in-out-sequences-of-generated-rest-api.jpg)]({{base_path}}/assets/img/Learn/in-out-sequences-of-generated-rest-api.jpg)
+
+     The following sample shows the generated API In-sequence for a POST method.
+
+     ``` java
```

Comment:
Should this be ```xml since this is an xml payload? Im not sure which formatting would be most appropriate, java or xml. Need to check which is better and make a call